packages/core-design-system/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@harnessio/core-design-system",
+  "version": "1.0.0",
```

Comment:
let's stay on `0.0.x` version until we can replace `gitness` references across the package

Reply:
Updated